rules/REQUEST-913-SCANNER-DETECTION.conf

```
@@ -55,6 +55,26 @@ SecRule REQUEST_HEADERS:User-Agent "@pmFromFile scanners-user-agents.data" \
     severity:'CRITICAL',\
     setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
+SecRule REQUEST_HEADERS:User-Agent "@pmFromFile ai-scrapers-user-agents.data" \
+    "id:913101,\
```

```suggestion
    "id:913110,\
```

Per https://github.com/coreruleset/coreruleset/blob/main/CONTRIBUTING.md#id-numbering-scheme, let's use `913110` as id.